GTG/gtk/ui/preferences.ui

```diff
@@ -0,0 +1,497 @@
+                        <child>
+                          <object class="GtkSwitch" id="preview_button">
+                            <property name="visible">True</property>
+                            <property name="valign">center</property>
+                            <signal handler="on_preview_button_toggled" name="state-set" swapped="no"/>
+                          </object>
+                        </child>
+                        <child>
+                          <object class="GtkBox">
+                            <property name="orientation">vertical</property>
+                            <property name="valign">center</property>
+                            <property name="vexpand">false</property>
+                            <property name="visible">true</property>
+                            <property name="margin-start">10</property>
+                            <child>
+                              <object class="GtkLabel" id="preview_title">
+                                <property name="label">Show description preview</property>
+                                <property name="visible">true</property>
+                                <property name="xalign">0.0</property>
+                                <attributes>
+                                  <attribute name="weight" value="bold"/>
+                                </attributes>
+                              </object>
+                            </child>
+                            <child>
+                              <object class="GtkLabel" id="preview_label">
+                                <property name="label">Show a short description preview next to each task in the task list</property>
+                                <property name="visible">true</property>
+                                <property name="xalign">0.0</property>
+                                <style>
+                                  <class name="dim-label"/>
+                                </style>
+                                <attributes>
+                                  <attribute name="scale" value="0.833333"/>
+                                </attributes>
+                              </object>
+                            </child>
+                          </object>
+                        </child>
+                      </object>
+                    </child>
+                    <child>
+                      <object class="GtkBox" id="bg_color_box">
+                        <property name="margin">20</property>
+                        <property name="margin-start">40</property>
+                        <child>
+                          <object class="GtkSwitch" id="bg_color_button">
+                            <property name="visible">True</property>
+                            <property name="valign">center</property>
+                            <signal handler="on_bg_color_toggled" name="notify::active" swapped="no"/>
```

Comment:
Why do you have here `notify::active` when up there you set `set-state`? Shoudn't they be the same?